dist/platform.h

```diff
@@ -44,10 +43,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 /*----------------------------------------------------------------------------
 *----------------------------------------------------------------------------*/
-#define INLINE static inline
```

Comment:
Why removal of this define?

Reply:
It isn't needed if we don't set the inline level.